flink-simple-tutorial/README.md

```diff
@@ -62,11 +62,11 @@ The Heap Monitoring Application has four structural components:
 
 ## Application main class
 
-A Flink application has to define a main class that will be executed on the client side on job submission. The main class will define the application pipeline that is going to be executed on the cluster.
+A Flink application as any regular Java program has to define a class that is the entry point for the execution. This entry point is executed on the client side, reads the local configuration and assembles the Flink pipeline which is then submitted to the cluster for execution.
 
-Our main class is the `HeapMonitorPipeline` which contains a main method like any standard Java application. The arguments passed to our main method will be determined by us when we use the Flink client. We use the  `ParameterTool` utility to conveniently pass parameters to our job that we can use in our operator implementations.
+In our tutorial this entry point is the `HeapMonitorPipeline` class, which contains a main function like any standard Java application. The program arguments passed to the `flink run` command are received directly by this main method. To be able to conveniently name these arguments on the command line and pass them in any order we use the `ParameterTool` utility. 
```

Comment:
In the tutorial, this entry point is the `HeapMonitorPipeline` class, which contains a main function like any standard Java application. The program arguments passed to the `flink run` command are received directly by this main method. To be able to conveniently name these arguments on the command line and pass them in any order, we use the `ParameterTool` utility.